packages/worker/src/core.ts

```diff
@@ -58,12 +69,40 @@ export class Core {
           : `http://${compiledServerOptions.address}`,
       },
       telemetryOptions,
+      logger: options.logger ?? new DefaultLogger("INFO")
     };
     const native = await promisify(newCore)(compiledOptions);
+    this.initLogPolling(options, native);
+
     return new this(native, compiledOptions);
   }
 
+  private static initLogPolling(options: CoreOptions, native: native.Core) {
+    if (options.logger != null) {
+      interval(3)
+        .pipe(takeUntil(this.shouldPollForLogs.pipe(filter((v) => v === false))))
```

Comment:
This doesn't support resuming, I can show you how to add that in rxjs.

Reply:
Does it need to be paused/resumed? It's on until core is gone, then on if you make a new core.